security-docs/trusted-root/Program-Requirements.md

```diff
@@ -114,13 +114,16 @@ Program.
 ### C. Revocation Requirements
 
 1.  CAs must have a documented revocation policy and must have the ability to revoke any certificate it issues.
-2.  CAs that issue Server Authentication certificates must support both of the following OCSP responder requirements:
-    1.  A minimum validity of eight (8) hours; a maximum validity of seven (7) days.
-    2.  The next update must be available at least eight (8) hours before the current period expires. If the validity is more than 16 hours, then the next update must be available at ½ of the validity period.
-3.  All certificates issued from a root CA must support either the CRL distribution point extension and/or AIA containing an OCSP responder URL.
+2. 	OCSP responder requirements:
+    a.	Minimum validity of eight (8) hours; Maximum validity of seven (7) days; and
+    b.	The next update must be available at least eight (8) hours before the current period expires. If the validity is more than 16 hours, then the next update must be available at ½ the validity period.
+3.	CRL recommendations when OCSP is not present:
+    a.	Should contain Microsoft-specific extension 1.3.6.1.4.1.311.21.4 (Next CRL Publish).
+    b.	New CRL should be available at the Next CRL Publish time.
+    c.	Maximum size of the CRL file (either full CRL or partitioned CRL) should not exceed 10M.
+    Note: The goal of section 3.C.3- CRL Recommendations when OCSP is not present is to provide coverage for end users in cases of mass revocation. 
```

Comment:
```suggestion
    > [!Note]
    > The goal of section 3.C.3- CRL Recommendations when OCSP is not present is to provide coverage for end users in cases of mass revocation. 
```
learn docs style is to use this syntax for 'notes'. 

there is another note in the article that should be changed as well, but you can do that in a different PR. thanks.